src/pages/TransactionsList.vue

```diff
@@ -84,26 +128,40 @@ const { width } = useWindowSize();
 
 const hashType = computed(() => (width.value < HASH_BREAKPOINT ? 'short' : 'full'));
 
-const table = useTable(http.fetchTransactions, { reversed: true });
 const { handleUnknownError } = useErrorHandlers();
 
-onMounted(async () => {
+const transactionsTable = useTable(http.fetchTransactions, { reversed: true });
+const instructionsTable = useTable(http.fetchInstructions);
+
+const listState = computed(() => ({
+  status: status.value,
+  kind: transactionType.value,
+}));
```

Comment:
No need to define this.

Reply:
I think it's better to keep this variable since later I will implement query parameters and it will be necessary to have such variable. Additionally, I will add table properties.